app/src/main/java/it/ministerodellasalute/immuni/workers/ExposureNotificationWorkers.kt

```diff
@@ -169,6 +169,108 @@ class RequestDiagnosisKeysWorker(
     }
 }
 
+class RequestDiagnosisEuKeysWorker(
```

Comment:
You may consider to avoid duplicate too much code here. This will lead to difficult debugging and maintanance in the future.